sourcegraph/sourcegraph
# [SG-43129] Accessibility: Closing search contexts dropdown should focus the dropdown trigger

### Description
When closing the search context dropdown menu with the Escape key, the close ("X") button, or the "Reset" button, nothing is focused or the search box is focused.

### Success Criteria
The search context dropdown button should be focused after closing the menu.

### Ref
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/43129)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-43129)

<img width="1049" alt="Screenshot 2022-10-19 at 14 28 46" src="https://user-images.githubusercontent.com/7220167/196678647-dacce342-bd70-482c-9e50-15f6c55df97b.png">


### Test Plan
1. Go to localhost:3080/search
2. Open the search context dropdown menu.
3. Use the `esc` key or click on the close `x` icon or click on the `Reset` link.
4. Focus should be returned to the Search Context Dropdown Menu as shown in the image below.

## App preview:

- [Web](https://sg-web-contractors-sg-43129.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
